frontend/src/app/logs/logs.component.html

```diff
@@ -1,13 +1,15 @@
-<div class="oib-padded-container">
+<div [ngClass]="{ 'oib-padded-container': this.scopeId == null && this.scopeType == null }">
```

Comment:
You can simplify by using !this.scopeId && !this.scopeType

And be careful, in JS you should use `===` instead of `==` ; or `!==` instead of `!=`